collab-service/src/db/index.ts

```diff
@@ -0,0 +1,119 @@
+import 'reflect-metadata';
```

Comment:
DB-related items can be removed since collab-service should not be on the data layer.